### DISCUSSION TITLE
Offline Cache available?

### DISCUSSION THREAD

**Discussion by Schlotti83** [Forceu/barcodebuddy#247] [Q&A]
Hi there!

I use BarcodeBuddy together with Grocy on my Unraid. Generally it works fine.
But I realised that I would like to scan items which I store in my basement without bringing them to the 4th Floor and down again.

So I tested it but realized, that it seems only to work, when it's connected (Wifi).

Is it a misusage or isn't this feature availabe right now?

Thanks a lot!

**Comment by Forceu** [comment]
In case you are using a barcode scanner, a lot of them offer an offline option where they store all barcodes until the scanner is connected again. Alternatively you could connect through a VPN b to your instance

**Comment by Schlotti83** [comment]
Sorry, I missed the Details.

I am using the mobile App on an Android Phone, with the integrated Cam. 
In this case, there is no cache?
